docs/en/Go-Client.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: global
+title: Go Client
+nickname: Go Client
+group: Clients
```

Comment:
@gpang I don't really see this group 
http://www.alluxio.org/docs/master/en/Getting-Started.html
can you double check if there is any change missing to make clients show up there?